build-support/bin/generate_github_workflows.py

```diff
@@ -414,6 +417,16 @@ def test_workflow_jobs(primary_python_version: str, *, cron: bool) -> Jobs:
     return jobs
 
 
+def upload_log_artifacts(name: str) -> Sequence[Step]:
```

Comment:
This can return a single `Step`, then you don't need to unpack everywhere with `*`.